<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carta347</class>
 <widget class="QDialog" name="carta347">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contenido Carta-Comunicación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cabecera:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cabeceralineEdit">
       <property name="text">
        <string>Estimados Señores/as</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cuerpo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="cuerpoplainTextEdit">
       <property name="plainText">
        <string>Al objeto de .... {cifra}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="desglosecheckBox">
     <property name="text">
      <string>Desglose trimestral</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cuerpo2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="cuerpoplainTextEdit_2">
       <property name="plainText">
        <string>Cifra que declararemos ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Pie del texto      :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pielineEdit">
       <property name="text">
        <string>Rogando que comprueben dicho dato,....</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Saludo de cierre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saludolineEdit">
       <property name="text">
        <string>Atentamente, </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Firma:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firmalineEdit">
       <property name="text">
        <string>Dpto. CONTABILIDAD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>carta347</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
